components/radio/radio.tsx

```diff
@@ -49,7 +49,7 @@ export default class Radio extends React.Component<RadioProps, any> {
         onMouseLeave={this.props.onMouseLeave}
       >
         <RcRadio {...this.props} className={classString} style={null} children={null} />
-        {children ? <span>{children}</span> : null}
+        {children !== undefined ? <span>{children}</span> : null}
```

Comment:
`children !== undefined && children !== null` doesn't cover all cases

In fact, booleans, null, and undefined are ignored, so just 
```js
{<span>{children}</span>}
``` 
still works fine.

ref: https://facebook.github.io/react/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored

btw in the checkbox component, the code is the same as `children !== undefined`